src/lib/utils/abstract_plugin.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <iostream>
+
+#include "storage/storage_manager.hpp"
+#include "types.hpp"
+#include "utils/singleton.hpp"
+
+namespace opossum {
+
+// This is necessary to make the plugin instantiable, it leads to plain C linkage to avoid
+// ugly mangled names. Use EXPORT in the implementation file of your plugin.
+#define EXPORT(Plugin)                                            \
```

Comment:
EXPORT could be anything. EXPORT_PLUGIN, maybe?

Reply:
Done!